packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -331,6 +334,27 @@ abstract class AnimatedWidgetBaseState<T extends ImplicitlyAnimatedWidget> exten
   void forEachTween(TweenVisitor<dynamic> visitor);
 }
 
+/// A base class for widgets with implicit animations that need to rebuild their
+/// Widget tree as the animation runs.
+///
+/// If you would rather respond to a running animation directly, use
+/// [ImplicitlyAnimatedWidgetState].
```

Comment:
api doc voice: prefer the passive voice, and never give direct instruction, merely suggest.

For example: `This class calls [build] each frame that the animation ticks. For a variant that does not rebuild each frame, consider subclassing [ImplicitlyAnimatedWidgetState] directly.`